Sources/Sentry/SentryThreadInspector.m

```diff
@@ -12,6 +18,36 @@
 
 @end
 
+typedef struct {
+    SentryCrashThread thread;
+    SentryCrashStackEntry stackEntries[MAX_STACKTRACE_LENGTH];
+    int stackLength;
+} ThreadInfo;
+
+// We need a C function to retrieve information from the stack trace in order to avoid
+// calling into async code while there is suspended threads.
```

Comment:
🥇 for adding this comment.
```suggestion
// We need a C function to retrieve information from the stack trace in order to avoid
// calling into not async-signal-safe code while there are suspended threads.
```